src/python/zquantum/qaoa/ansatzes/warm_start_ansatz_test.py

```diff
@@ -0,0 +1,153 @@
+from zquantum.core.interfaces.ansatz_test import AnsatzTests
+from zquantum.core.circuit import Circuit, Gate, Qubit
+from zquantum.core.utils import compare_unitary
+from zquantum.core.openfermion import change_operator_type
+from .warm_start_ansatz import WarmStartQAOAAnsatz
+from openfermion import QubitOperator, IsingOperator
+import pytest
+import numpy as np
+import sympy
+
+
+class TestQAOAFarhiAnsatz(AnsatzTests):
+    @pytest.fixture
+    def thetas(self):
+        return np.array([0.5, 0.5])
+
+    @pytest.fixture
+    def ansatz(self, thetas):
+        cost_hamiltonian = QubitOperator((0, "Z")) + QubitOperator((1, "Z"))
+        return WarmStartQAOAAnsatz(
+            number_of_layers=1,
+            cost_hamiltonian=cost_hamiltonian,
+            thetas=thetas,
+        )
```

Comment:
I think it's important to test the ansatz with at least 2 layers in addition to just the 1 layer ansatz

Reply:
Here you are: https://github.com/zapatacomputing/z-quantum-qaoa/pull/31/commits/eb41b606cd948b9b4f8e664ca8165d7e33742fa7